docs/outputs.rst

```diff
@@ -4,97 +4,151 @@ Outputs of tedana
-Filename                  Content
-======================    =====================================================
-t2sv.nii                  Limited estimated T2* 3D map.
-                          The difference between the limited and full maps
-                          is that, for voxels affected by dropout where
-                          only one echo contains good data, the full map
-                          uses the single echo's value while the limited
-                          map has a NaN.
-s0v.nii                   Limited S0 3D map.
-                          The difference between the limited and full maps
-                          is that, for voxels affected by dropout where
-                          only one echo contains good data, the full map
-                          uses the single echo's value while the limited
-                          map has a NaN.
-ts_OC.nii                 Optimally combined time series.
-dn_ts_OC.nii              Denoised optimally combined time series. Recommended
-                          dataset for analysis.
-lowk_ts_OC.nii            Combined time series from rejected components.
-midk_ts_OC.nii            Combined time series from "mid-k" rejected components.
-hik_ts_OC.nii             High-kappa time series. This dataset does not
-                          include thermal noise or low variance components.
-                          Not the recommended dataset for analysis.
-comp_table_pca.txt        TEDPCA component table. A tab-delimited file with
-                          summary metrics and inclusion/exclusion information
-                          for each component from the PCA decomposition.
-mepca_mix.1D              Mixing matrix (component time series) from PCA
-                          decomposition.
-meica_mix.1D              Mixing matrix (component time series) from ICA
-                          decomposition. The only differences between this
-                          mixing matrix and the one above are that
-                          components may be sorted differently and signs of
-                          time series may be flipped.
-betas_OC.nii              Full ICA coefficient feature set.
-betas_hik_OC.nii          High-kappa ICA coefficient feature set
-feats_OC2.nii             Z-normalized spatial component maps
-comp_table_ica.txt        TEDICA component table. A tab-delimited file with
-                          summary metrics and inclusion/exclusion information
-                          for each component from the ICA decomposition.
-======================    =====================================================
+========================================   =====================================================
+Filename                                   Content
+========================================   =====================================================
+*_desc-limited_t2s.nii.gz                  Limited estimated T2* 3D map.
```

Comment:
Suffices `t2s` and `s0` are pending integration into BIDS Derivatives.

Reply:
The discussion is in https://github.com/bids-standard/bids-specification/issues/127

Reply:
It sounds like we're good to go with T2Starmap and S0map. Should I update this PR with those suffixes?